doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
+To start the community discussion around potential gaps in the body of existing standards around the Cloud Infrastructure (NFV Infrastracture) networking, we looked into the ETSI NFV Architectural Framework. We have identified a number of areas that, in our view, need to be addressed (see below for the initial list). As usual, in our further efforts, the community will need to decide where and how these items should be addressed.
 
-For CNTT, a strategy is needed that affords Operators the performance, flexibility, availability, maintainability and scalability their business requires, yet doesn't require OPNFV to design, manage and test prohibitive numbers of networking solutions.
- 
-**Some points for CNTT to consider when contemplating recommendations:**
-   > * Despite large variances in implementation, the spectrum of networking capabilities ultimately delivered to Workloads is comparatively narrow
-   > * Standard CNTT methodology (i.e. normalize interfaces, APIs, capabilities and behaviors at the reference points) applies well to networking, helping mitigate the need to be overly prescriptive about implementation  
-   > * CNTT/OPNFV MUST provide a functional networking solution for the RI, and in support of RC  
-   > * CNTT/OPNFV potentially does NOT have to provide a production networking solution for RI  
-   > * CNTT/OPNFV is NOT planning to provide a production compute solution for RI
+•	ETSI NFV does not have a separation between HW Infrastructure Management and SW Virtualisation Management and by that the cardinality of having multiple CaaS and IaaS layers on top of a shared HW Infrastructure Layer cannot be expressed
 
+•	ETSI NFV lacks a description of the reference points in between the SW Virtualisation Layers and the HW Infrastructure Layer denoted as Vl-Ha and by that cannot express Packet Flows, Control/Status Interfaces and Management Interfaces between them
 
-<a name="5.1.1"></a>
-### 5.1.1 Executive Summary
+•	ETSI NFV model does not explicitly call out an HW Abstraction layer on top of the HW resources and by that many SW Virtualisation Layers act upon the physical HW components in a non-portable way that do not allow an HW Infrastructure to compose abstracted HW resources from pools of individual HW components
+
+•	ETSI NFV lack of spelled out HW Abstraction does not enable a clear way to abstract and describe the technical possibilities that HW layer networking separation, encapsulation, acceleration, etc. can be implemented either in a HW Infrastructure Managed SmartNIC on the Server units or in the shared underlay switching fabric i.e. on the physical Switching Units 
+
+•	ETSI NFV model does not include any reference to SDN (Software Define Network) controller(s) and relevant integration points into NFVI and MANO 
+
+•	ETSI NFV model does not have a way to enable programmable forwarding planes in the HW layer controlled from higher layers of virtualization managers, orchestrators or Network Functions
```

Comment:
As far as I understand, this is part of the interface between Network Functions and Networking HW layer. NFV’s goal was not to create a standardized interface for this. Different flavours of network intensive resources could provide different interfaces.

Reply:
@walterkozlowski Is this different than for a fabric using distributed control? AFAIK ETSI doesn't do this for distributed, either. Are you envisioning something like the vRouter interface? Thx, -Mark

Reply:
@ulikleber 
- The preference (ideal case) is that the VNF/CNF is completely unaware and unconcerned with any sort of deployment time programmable forwarding plane acceleration function. Where this can be achieved there is some sort of Infrastructure orchestrator in the MANO section that must have an interface to the Infrastructure part (HW or SW) where this programmability could be controlled. One example could be OVS acceleration.
- In many cases it is however hard to hide the Infrastructure programmability and it will unfortunately shine through to the VNF/CNF, making it necessary to use interfaces in between the Infrastructure (HW or SW) into the VNF/CNF. One example could be VNF/CNF usage of SR-IOV.
-- In both these cases CNTT ought to / need to have a more detailed specification then what ETSI NFV have today to simplify for the Operators in their quest to deploy suitable VNF/CNF and build / source suitable NFVI functionality and characteristics
-- Agree on that different flavours of servers and different flavours of networking resources could provide different interfaces, but our target here is to abstract as much of that as possible and only let the most necessary differences shine through

Reply:
@markshostak  I could not see that the realization of a centralized controller or a distributed controller should be visible in the interfaces in the reference points outside of the fabric.